src/test/feature/ManagementTool/test_hawq_register.cpp

```diff
@@ -341,15 +341,30 @@ TEST_F(TestHawqRegister, TestIncorrectYaml) {
   EXPECT_EQ(1, Command::getCommandStatus("hawq register -d " + (string) HAWQ_DB + " -c " + filePath + "incorrect4.yml xx"));
   EXPECT_EQ(1, Command::getCommandStatus("hawq register -d " + (string) HAWQ_DB + " -c " + filePath + "incorrect5.yml xx"));
   EXPECT_EQ(1, Command::getCommandStatus("hawq register -d " + (string) HAWQ_DB + " -c " + filePath + "incorrect6.yml xx"));
+  EXPECT_EQ(1, Command::getCommandStatus("hawq register -d " + (string) HAWQ_DB + " -c " + filePath + "incorrect8.yml xx"));
 }
 
-TEST_F(TestHawqRegister, TestCreateExistedTable) {
+TEST_F(TestHawqRegister, TestDismatchFileNumber) {
+  SQLUtility util;
+  string filePath = util.getTestRootPath() + "/ManagementTool/";
+  EXPECT_EQ(1, Command::getCommandStatus("hawq register -d " + (string) HAWQ_DB + " -c " + filePath + "incorrect7.yml xx"));
+}
+
+TEST_F(TestHawqRegister, TestUsage2Behavior2) {
   SQLUtility util;
   util.execute("drop table if exists t10;");
-  util.execute("create table t10(i int) with (appendonly=true, orientation=row) distributed by (i);");
```

Comment:
done. I modify the name of test yml file, and some of the cases. I think I will modify table names of all cases in the following pull request.